src/openforms/formio/dynamic_config/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class FormioDynamicConfigApp(AppConfig):
```

Comment:
Nu snap ik wat je gisteren bedoelde met laat dat register patroon in global filetypes maar zitten :)

Reply:
ja, de boilerplate heeft wel even wat voeten in de aarde :grimacing: